### PR TITLE
[Documentation] Sylius Extensions

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/extensions/index.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Sylius Extensions
+=================
+
+Sylius as a platform has a lot of space for various customizations and extensions. It aims to provide a simple schema for
+developing extensions(plugins). Anything you can imagine can be implemented and added to the Sylius framework as an extension.
```
Comment (venyii) [inline docs/extensions/index.rst]: extensions(plugins) -> extensions (plugins)

File: docs/extensions/index.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Exemplary features may be: Social media buttons, newsletter, wishlists etc.
+
+Extension Development Flow
+--------------------------
+
+Since Sylius is an open-source platform, there is a certain flow in order for the extension to become officially adopted by the community.
+
+**1.** Develop the extension using :doc:`the official Extension Development guide </extensions/creating-extension>`.
+
+**2.** Send it to the project maintainers. It can be via email, Slack or even GitHub issues.
```
Comment (venyii) [inline docs/extensions/index.rst]: Would it make sense to add links to these contact methods? Where to find more info about the Slack channel oder sth. like that?

Comment (pamil) [inline docs/extensions/index.rst]: And we should not mention GitHub issues here 💃

File: docs/extensions/list.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Sylius Extensions List
+======================
+
+.. note::
+
+    This section will grow whenever a new extension will be provided by the Sylius open-source community!
+
+The current list of Sylius Extensions presents as follows:
+
+* `Lakion\SyliusCmsBundle`_
```
Comment (pamil) [inline docs/extensions/list.rst]: Shouldn't we use inline links here? There won't be the same extension mentioned twice and less lines = better :>

**Comment by pjedrzejewski** [comment]
Thanks Magda! :)

**Pull request merged**